packages/x-domain/test/actions.spec.ts

```diff
@@ -71,75 +73,101 @@ const deposit = async (amount, value) => {
   const receipt = await watcher.getL2TransactionReceipt(msgHash)
 }
 
-describe('SimpleStorage', async () => {
+const withdraw = async (value) => {
```

Comment:
What purpose does having the `withdraw` and `deposit` functions serve? It looks like they are just used once